pkg/operator/controller/ingress/status_test.go

```diff
@@ -279,26 +289,6 @@ func TestComputeIngressDegradedCondition(t *testing.T) {
 			// Just use the one minute retry duration for this degraded condition
 			expectAfter: time.Minute,
 		},
-		{
-			name: "pods not scheduled for <10m",
-			conditions: []operatorv1.OperatorCondition{
-				cond(IngressControllerPodsScheduledConditionType, operatorv1.ConditionFalse, "", clock.Now().Add(time.Minute*-9)),
-			},
-			expectIngressDegradedStatus: operatorv1.ConditionFalse,
-			expectRequeue:               true,
-			// Grace period is 10 minutes, subtract the 9 minute spoofed last transition time
-			expectAfter: time.Minute,
-		},
-		{
-			name: "pods not scheduled for >10m",
-			conditions: []operatorv1.OperatorCondition{
-				cond(IngressControllerPodsScheduledConditionType, operatorv1.ConditionFalse, "", clock.Now().Add(time.Minute*-31)),
-			},
-			expectIngressDegradedStatus: operatorv1.ConditionTrue,
-			expectRequeue:               true,
-			// Just use the one minute retry duration for this degraded condition
-			expectAfter: time.Minute,
-		},
```

Comment:
Do we no longer have a grace period for which pods can remain unscheduled and not yet considered unavailable?

Reply:
There is no longer a grace period for the `PodsScheduled` status condition, because the `PodsScheduled` status condition no longer even exists after this PR.  

The point of this PR is to remove the `PodsScheduled` status condition because it was spuriously indicating a condition (some pods weren't scheduled) as a problem when really it wasn't necessarily a problem at all (the not-scheduled pods might actually be pods that are waiting to be garbage-collected).  However, the operator still reports the `DeploymentReplicasMinAvailable` status condition (which has a 60-second grace period, before and after this PR), which conveys the information that really does matter (whether there are _enough_ pods that are scheduled, running, and ready).

Reply:
So will this automatically continue to reconcile while there are not enough pods scheduled?

Reply:
Yeah, if there are not _enough available_ pods, then `DeploymentReplicasMinAvailable` will have status `False`, and the grace logic will queue reconciliation for the end of the condition's grace period.  This is another way in which `PodsScheduled` is superfluous:  `DeploymentReplicasMinAvailable` has a shorter grace period than `PodsScheduled` had, so if `DeploymentReplicasMinAvailable` is `False`, its grace period already would have triggered reconcilation (and still does trigger it) before `PodsScheduled`'s grace period would have triggered reconciliation.